src/Datadog.Trace.ClrProfiler.Managed/ScopeFactory.cs

```diff
@@ -1,5 +1,6 @@
 using System;
 using System.Data;
+using Datadog.Trace.ClrProfiler.AutoInstrumentation.RabbitMQ;
```

Comment:
I figured we'd want to consolidate the Scope creation (for both CallSite and CallTarget) somewhere. Is this class the best candidate to hold the method?

Reply:
I wonder why do you need this using in the ScopeFactory, it's because the RabbitMQ constants class?

Reply:
Yes, it's because of the RabbitMQ constants class

Reply:
Not asking for any change in this PR, but I'm wondering if grouping scope creating for all the disparate instrumentations makes sense, or it this something that should be part of each integration? There are only a few that are shared like http and adonet clients, but the rest have nothing in common.

Reply:
You're right it should be for each integration. I was trying to figure out the best place to share the scope creation between CallSite and CallTarget. I'll move the RabbitMQ scope creation back to the original RabbitMQIntegration and just have the CallTarget instrumentation use that